packages/devextreme/js/__internal/ui/m_tag_box.ts

```diff
@@ -798,6 +806,7 @@ const TagBox = (SelectBox as any).inherit({
       })
       .done((data, extra) => {
         this._isDataSourceChanged = false;
+        this._newDataSource = false;
```

Comment:
The `_newDataSource` doesn't say that it stores value with boolean type.

Reply:
```suggestion
        this._isDataSourceOptionChanged = false;
```

Reply:
renamed to `_isDataSourceOptionChanged`